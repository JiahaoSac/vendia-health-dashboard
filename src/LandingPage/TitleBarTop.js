import React from 'react';
import { Typography } from '@mui/material';
import styled from 'styled-components';

function TitleBarTop() {
    return (
        <STYLE_1>
           <h1 style={{marginTop: '0px'}}>Welcome to Vendia Health Dashboard</h1>
        </STYLE_1> 
    );
};

const STYLE_1 = styled.div`

    background-image: linear-gradient(144deg,#adffd0, #8afff7 50%,#a3d9ff);
    text-align: center;
    font-family: 'Ubuntu', sans-serif;

`

export default TitleBarTop;
