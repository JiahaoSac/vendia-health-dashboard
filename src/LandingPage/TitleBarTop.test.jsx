import TitleBarTop from "./TitleBarTop.js";
import renderer from "react-test-renderer";

it("Title appears", () => {
  const component = renderer.create(<TitleBarTop />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
