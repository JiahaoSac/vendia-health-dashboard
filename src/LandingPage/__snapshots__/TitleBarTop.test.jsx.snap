// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title appears 1`] = `
<div
  className="sc-bczRLJ eJCTEJ"
>
  <img
    alt="website logo"
    src="white.svg"
    style={
      Object {
        "height": 65,
        "width": 199,
      }
    }
  />
  <h1
    style={
      Object {
        "color": "white",
        "fontSize": "1.5rem",
        "marginTop": "0px",
      }
    }
  >
    Health Dashboard
  </h1>
</div>
`;
